### PR TITLE
Add yargs to checkoutCore, exit with error code on error, https clone

### PR DESCRIPTION
**Pull request by paolobueno**
## Motivation
Fix travis build, add prettier UX to cli

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -7,8 +7,8 @@ services:
 - docker
 - mongodb
 script:
-- npm run bootstrap
-- npm test
+- npm run bootstrapCI
+- npm run testCI
 cache:
   directories:
     - node_modules
```
Comment (wtrocki) [inline .travis.yml:14]: Can we add more directories. Build will be slightly faster.

Comment (wtrocki) [reply .travis.yml:14]: see: https://github.com/feedhenry-raincatcher/raincatcher-angularjs/pull/31

**Review by wtrocki — DISMISSED** [review]
Reviewed. Verified command line tools locally.

File: .travis.yml (hunk 1 above)
Comment (paolobueno) [reply .travis.yml:14]: Done

**Review by wtrocki — APPROVED** [review]

**Pull request merged**